software/duet_config_files/duet2/rrf3/sys/homeu.g

```diff
@@ -1,8 +1,22 @@
 ; Home U Axis
 
+; Assumes the tool changer is in 1 of 2 possible states.
+; 1. No tool attached (U=0), usually the result of the tool being properly parked.
+; 2. Tool attached (U=?), usually the result of the machine being powered powered off with a tool still active.
+
+G90                     ; Set absolute mode
+G92 U0                  ; Define current position as 0 to enable move without homing
+G1 U2 H2 F5000          ; Move the axis 2deg to back it off of the unlocked switch
+G1 U170 H1 F5000        ; Move the axis to 170deg, or until it hits an endstop
+M400                    ; Make sure moves are complete
+
+if abs(move.axes[3].userPosition - 170) > 1
+    M84 U
+    M291 R"Intervention Required" P"A tool has been detected! Please remove and return to post. Press OK to continue..." S2
```

Comment:
As is, this error will trigger if the Twist Lock is not horizontal by default, regardless of whether or not it has tools. The machine is pretty rigorous about leaving an empty carriage with a horizontal twist lock, but add to this comment to tell the user to restore the twist lock to the horizontal unlocked position. Something like this:
```Please remove the tool, return it to its post, and restore the twist lock to its unlocked (horizontal) position.```

I can add to the instructions that, upon first powerup, builders should put the Twist Lock in the unlocked position first too. That way they don't get greeted with this error from the very first time they try to use the machine.